solutions/13.js

```diff
@@ -13,6 +12,22 @@ const solution = (str) => {
 	return true;
 };
 
+// Maricris Bonzo: @seemcat
+const solution1 = (string) => {
+  let indexLength = string.length - 1;
+  let reverseString = '';
+
+  for(i = 0; i <= indexLength; i++){
+    reverseString = string[i] + reverseString;
```

Comment:
This does not look like it will reverse the string.  Can you `console.log(reverseString)` to make sure?  and run `npm run test`.